server/src/main/java/org/elasticsearch/script/AbstractFieldScript.java

```diff
@@ -31,7 +31,7 @@ public abstract class AbstractFieldScript extends DocBasedScript {
      */
     public static final int MAX_VALUES = 100;
 
-    static <F> ScriptContext<F> newContext(String name, Class<F> factoryClass) {
+    protected static <F> ScriptContext<F> newContext(String name, Class<F> factoryClass) {
```

Comment:
This come from an unused class, let's remove everything? we can revisit later.

Reply:
Agreed. I also remember we discussed with the painless team the option of building a completely different fields API for script for this, so none of the prototype code here is of value. I created an issue to discuss that at https://github.com/elastic/elasticsearch/issues/90182

Reply:
Fixed in https://github.com/elastic/elasticsearch/pull/89216/commits/3fa56032243e566da4f8f4c70beb7f964531a1ad